cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -203,6 +203,7 @@ impl LockedPackageCatalog {
     ) -> Self {
         // unpack package to avoid missing new fields
         let catalog::PackageResolutionInfo {
+            catalog: _,
```

Comment:
nonblocking: should we put this in the lockfile? Is this going to be required at some point?

Reply:
Maybe, but I think there's a number of upcoming changes that will be needed.  I foresee a discussion and PR focused on just that in the very near future.